### PR TITLE
try to improve README.md for new contributors

### PR DESCRIPTION
**Pull request by lldelisle**
@mvdbeek would you mind to fill the `planemo workflow_test_on_invocation` part (I did not managed to make this command work...).

### CODE REVIEW

**Comment by mvdbeek** [comment]
What didn't work ?

File: workflows/README.md
Comment (mvdbeek) [inline workflows/README.md:145]:
```suggestion
planemo workflow_test_on_invocation --galaxy_url <your_prefered_galaxy_server --galaxy_user_key <your_api_key> <workflow-tests.yml> <invocation_id>
```

**Comment by lldelisle** [comment]
I did:
```bash
$ planemo test --galaxy_url https://usegalaxy.eu --galaxy_user_key $(cat ~/switchdrive/galaxy_eu.txt) se-wgs-variation.ga 
waiting for invocation 8704148c5c8a4c71
collecting outputs to directory /tmp/tmpe5pn36w9
galaxy.tool_util.verify INFO: ## files diff on '/home/ldelisle/Documents/mygit/iwc/workflows/sars-cov-2-variant-calling/sars-cov-2-se-illumina-wgs-variant-calling/test-data/final_snpeff_annotated_variants.vcf' and '/tmp/tmpvgvlihhnfinal_snpeff_annotated_variants.vcf': lines_diff = 6, found diff = 362
None
galaxy.tool_util.verify INFO: ## files diff on '/home/ldelisle/Documents/mygit/iwc/workflows/sars-cov-2-variant-calling/sars-cov-2-se-illumina-wgs-variant-calling/test-data/final_snpeff_annotated_variants.vcf' and '/tmp/tmp3ydoisp4final_snpeff_annotated_variants.vcf': lines_diff = 6, found diff = 362
None
```
And then
```bash
$ planemo workflow_test_on_invocation --galaxy_url https://usegalaxy.eu --galaxy_user_key $(cat ~/switchdrive/galaxy_eu.txt) se-wgs-variation-tests.yml 8704148c5c8a4c71
waiting for invocation 8704148c5c8a4c71
collecting outputs to directory /tmp/tmpeq9cw2ru
bioblend WARNING: GET: error 404: b'{"err_msg": "No such workflow found.", "err_code": 404001}', 9 attempts left
bioblend WARNING: GET: error 404: b'{"err_msg": "No such workflow found.", "err_code": 404001}', 8 attempts left
bioblend WARNING: GET: error 404: b'{"err_msg": "No such workflow found.", "err_code": 404001}', 7 attempts left
bioblend WARNING: GET: error 404: b'{"err_msg": "No such workflow found.", "err_code": 404001}', 6 attempts left
bioblend WARNING: GET: error 404: b'{"err_msg": "No such workflow found.", "err_code": 404001}', 5 attempts left
bioblend WARNING: GET: error 404: b'{"err_msg": "No such workflow found.", "err_code": 404001}', 4 attempts left
bioblend WARNING: GET: error 404: b'{"err_msg": "No such workflow found.", "err_code": 404001}', 3 attempts left
```

I was wondering if it was a bug or if it was me who did not know how to use it...

File: workflows/README.md
Comment (lldelisle) [inline workflows/README.md:147]: @mvdbeek can you confirm?

**Comment by lldelisle** [comment]
Do you want to wait for the release of planemo before merging this?

**Comment by mvdbeek** [comment]
I released planemo last week, 0.75.19 should work

**Review by mvdbeek — APPROVED** [review]

**Comment by mvdbeek** [comment]
Attention: deployment skipped!

https://github.com/galaxyproject/iwc/actions/runs/7212073352

**Pull request merged**
